src/renderer/pages/TorrentListPage.js

```diff
@@ -220,6 +196,29 @@ module.exports = class TorrentList extends React.Component {
           {downloadIcon}
         </i>
       )
+
+      if (TorrentPlayer.isPlayableTorrentSummary(torrentSummary)) {
+        playButton = (
+          <i
+            key='play-button'
+            title={playTooltip}
+            className={'button-round icon play ' + playClass}
```

Comment:
Fixed in https://github.com/feross/webtorrent-desktop/commit/83901eecbaa8441d6dd0d07d104d168a9a5df17e